alephium/contracts/upgradable.ral

```diff
@@ -0,0 +1,23 @@
+Abstract Contract Upgradable() extends Constants() {
+  fn parseConractUpgrade(payload: ByteVec) -> (ByteVec, ByteVec, ByteVec, ByteVec) {
```

Comment:
Since this is a pure function and it's only called in migration, we could move it to `TokenBridgeFactory` so that the code will not be loaded in most cases.